app/components/History/State.tsx

```diff
@@ -0,0 +1,63 @@
+import theme from '@/styles/theme';
+import styled from 'styled-components';
+
+interface StateBoxProps {
+  data: { count: number; title: string }[];
+  stateHandler: (idx: number) => void;
+  selectState: number[];
+  type: string;
+  title: string;
+}
+const StateBox = ({
+  data,
+  stateHandler,
+  selectState,
+  type,
+  title,
+}: StateBoxProps) => {
+  return (
+    <StateWrapper>
+      {data.map((data, idx) => (
+        <State
+          key={idx}
```

Comment:
```suggestion
          key={data.title}
```

key를 index로 쓰는 것을 추천하지 않습니다.
배열의 순서는 고정이 아니라서 언제든지 변경될 수 있고, 또 항목 순서나 아이템을 수정할 때 문제가 발생할 수 있습니다.

참고: https://ko.react.dev/learn/tutorial-tic-tac-toe#picking-a-key